frontend/src/components/pages/CourseOverview.tsx

```diff
@@ -1,25 +1,67 @@
 import React from "react";
-import { Box, Flex, VStack } from "@chakra-ui/react";
+import { useHistory, useParams } from "react-router-dom";
+import { ChevronLeftIcon } from "@chakra-ui/icons";
+import { Box, Button, Flex, Text, VStack } from "@chakra-ui/react";
+import { useQuery } from "@apollo/client";
 import Banner from "../learner/Banner";
 import ModuleLessonCount from "../learner/ModuleLessonCount";
+import { GET_COURSE } from "../../APIClients/queries/CourseQueries";
+import { COURSE_OVERVIEW_BASE_ROUTE } from "../../constants/Routes";
+import { CourseOverviewParams } from "../../types/CourseOverviewTypes";
 
 const ViewCourse = (): React.ReactElement => {
+  const history = useHistory();
+  const { courseID }: CourseOverviewParams = useParams();
+  const { data: courseData } = useQuery(GET_COURSE, {
+    variables: {
+      id: courseID,
+    },
+  });
+
+  const onReturnToCourseOverviewClick = () => {
+    history.push(COURSE_OVERVIEW_BASE_ROUTE);
+  };
+
+  const getLessonCount = () => {
+    let lessonCount = 0;
+
+    courseData?.course?.modules?.forEach((moduleContent: any) => {
```

Comment:
To fix the linter error, you could probably import `ModuleType` from `ModuleEditorTypes.ts` (yeah, a misnomer of a file name... but whatever)